spaceinvaders.py

```diff
@@ -70,15 +70,18 @@ def update(self, keys, *args):
 
 
 class Enemy(sprite.Sprite):
-    def __init__(self, row, column):
-        sprite.Sprite.__init__(self)
+    row_scores = {0: 30, 1: 20, 2: 20, 3: 10, 4: 10}
+
+    def __init__(self, row, column, *groups):
         self.row = row
         self.column = column
+        super(Enemy, self).__init__(*groups)
```

Comment:
Should this be first?

Reply:
There are params in super-constructor of `Sprite` class. I thought, it's ok to use it.
`sprite.py`:
```
class Sprite(object):
...
    def __init__(self, *groups):
        self.__g = {} # The groups the sprite is in
        if groups:
            self.add(*groups)
...
    def add(self, *groups):
...
        for group in groups:
...
                    group.add_internal(self)
...
```
The `Enemy.row` and `Enemy.column` are used in `Sprite.__init__` and needs to go first.
```
EnemyGroup.add_internal(...):
    def add_internal(self, *sprites):
...
        for s in sprites:
            self.enemies[s.row][s.column] = s
```
Yes, not clear enough. May be, add some comment or ever eliminate `(*groups)` usage?

Reply:
A little bit of a feng shui in the 1e87cac `Replace unobvious Sprite(*groups) w Sprite.add(*groups)`.